drivers/hmis/app/graphql/types/hmis_schema/has_enrollments.rb

```diff
@@ -16,13 +16,25 @@ def enrollments_field(name = :enrollments, description = nil, type: [Types::Hmis
           default_field_options = { type: type, null: false, description: description }
           field_options = default_field_options.merge(override_options)
           field(name, **field_options) do
+            argument :sort_order, HmisSchema::EnrollmentSortOption, required: false
             instance_eval(&block) if block_given?
           end
         end
       end
 
-      def resolve_enrollments(scope = object.enrollments, _user: current_user)
-        scope
+      def resolve_enrollments_with_loader(association_name = :enrollments, **args)
+        load_ar_association(object, association_name, scope: apply_enrollment_arguments(Hmis::Hud::Enrollment.all, **args))
```

Comment:
calling `Hmis::Hud::Enrollment.all` makes me nervous.  Should this be `Hmis::Hud::Enrollment.visible_to(current_user)` or something similar?

Reply:
Looks like clients and organizations are limited in the apply_enrollment_arguments method, maybe this should just be applied there for enrollments as well?
There is definitely a case where you can see a client but not _all_ of their enrollments.

Reply:
We can pass another scope is that's more appropriate. Because we're loading an association, we have to call a scope on `Hmis::Hud::Enrollment`, which serves as the base scope to load the association from. We could use `visible_to` instead of `all` if that makes you less nervous.

Reply:
as long as visibility is being taken into account somewhere in the chain, I'm happy.  Just wasn't seeing that for enrollments.  If we haven't implemented Enrollment.visible_by (or visible_to, or whatever the current version is) then we should, and it should just join project and merge in project visibility.

Reply:
just pushed a stub `Enrollment.viewable_by`.  Let me know if you need something different.

Reply:
@eanders Thanks! I switched to using that and pushed. Take a look and let me know if it looks good to you